### DISCUSSION TITLE
PostgreSQL v16 supported?

### DISCUSSION THREAD

**Discussion by prozac64** [requarks/wiki#6746] [Help / Questions] [answered]
I am trying to keep all platform components to the highest available level.
Wiki.js: 2.5.300
Node.js: 20.7

And now i want to upgrade PostgreSQL to version 16. 
Any experience with this version? Are there issues to be expected?

**Comment by aapwikip** [comment]
I see v3.0 coming out with PostgreSQL v16

**Comment by NGPixel** [comment, marked as answer]
There should be no issues with Postgres 16.

**Comment by prozac64** [comment]
Thnx, wil post my experiences right her.
